### ISSUE TITLE
Bundle build not refreshing / updating html files when changed

### ISSUE THREAD

**Issue by abhipanda** [jspm/jspm-cli#1848]
Hi Guy and team,

I am running jspm 0.17.0-beta.15 and using the latest fast build bundle. Running the command

`jspm bundle src/app/app.js app.build.js -wid`

works fine  for JS files but facing issues when changing htmls. Its not updating the module.html tagged with plugin-text

```
 File src/app/modules/my-project/module/module.html changed, rebuilding...
 Building the bundle tree for src/app/app.js...

app/modules/my-project/module/module.html!github:systemjs/plugin-text@0.0.8/text.js
```

Thanks,
Abhi

**Comment by abhipanda** [comment]
Solution:

Change 

`builder.invalidate(toFileURL(file));`

to

`builder.invalidate(toFileURL(file)+'*');`

**Comment by jonricaurte** [comment]
@abhipanda I had to reinstall plugin-jade. Try reinstalling.

**Comment by guybedford** [comment]
Released in 0.17.0-beta.18.
